luautils/lua.go

```diff
@@ -35,16 +33,11 @@ func NewProcessor(path string) *Processor {
 			panic(err)
 		}
 	}
-
-	processor := &Processor{
-		L:        L,
-		BasePath: path,
-	}
-
-	// register custom modules
-	RegisterEncodingModule(processor, L)
-	RegisterFSModule(processor, L)
+	p := &Processor{BasePath: path}
```

Comment:
are we sure this is safe to just return the lua state?  I don't know if it'll gc safely for instance.

Reply:
We have to remember to add:
```
defer L.Close()
```